### ISSUE TITLE
Saving your AFS-planning

### ISSUE THREAD

**Issue by noobla11** [hydrocode-de/integra-frontend#176]
We discussed this multiple times but did not really agree on a solution I think? We talked about it again today and Zoe proposed a downloadable log-file which can be uploaded again. Maybe .json or .xml. I think this idea came up earlier as well. Is this possible? Otherwise we need to think about a different solution.

If possible we would like to have a pop-up where the user is asked if he wants to start planning a new system or if he wants to upload an existing file.

**Comment by mmaelicke** [comment]
Yeah, there is a lot already there, but deactivated and hidden until the data model is final. But thanks for pointing out.

**Comment by noobla11** [comment]
Changed this to medium, because I realised it might be less important to offer a saving option at the moment? The most important is the functionality of the app and to show which info is summarised I guess.
